CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# to-be-released
+
+## Changed
+
+- [BREAKING] Internal representation of hash schemas was changed to be a simple list of key types (flash-gordon)
+  `Dry::Types::Hash#with_type_transform` now yields a key type instead of type + name:
+  ```ruby
+  Dry::Types['stirct.hash'].with_type_transform { |key| key.name == :age ? key.required(false) : key }
+  ```
+
+## Added
+
+- Optional keys for schemas can be provided with ?-ending symbols (flash-gordon)
+  ```ruby
+  Dry::Types['stirct.hash'].schema(name: 'strict.string', age?: 'strict.integer')
```

Comment:
@flash-gordon FYI, there's a `stirct` typo across all of these CHANGELOG entries that you'll want to fix :)